src/Component/AutoMapper/Tests/Fixtures/SymfonyUlidUser.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Jane\Component\AutoMapper\Tests\Fixtures;
+
+use Symfony\Component\Uid\Ulid;
+
+class SymfonyUlidUser
+{
+    /**
+     * @var Ulid
+     */
+    private $ulid;
```

Comment:
```suggestion
    public Ulid $ulid;
```

why docblock comments?

Reply:
Because Jane supports PHP 7.2 so we can't use typed properties atm :grimacing: